### ISSUE TITLE
machine accessibility

### ISSUE THREAD

**Issue by sbyim** [elexis-eu/edie#12]

**Comment by jmccrae** [comment]
@sbyim Can you add more description or close this issue?

**Comment by sbyim** [comment]
@jmccrae Yes, we have briefly discussed this issue in a meeting as far as i can remember, and meant it won't be easily possible, so let's close it.
